src/Inciter/Discretization.C

```diff
@@ -42,9 +42,9 @@ Discretization::Discretization(
   const std::unordered_map< std::size_t, std::size_t >& filenodes,
   const tk::UnsMesh::EdgeNodes& edgenodes,
   int nchare,
-  std::size_t nbfac,
+  std::size_t tnbfac,
   const std::map< int, std::vector< std::size_t > >& bface,
-  const std::map< int, std::vector< std::size_t > >& belem ) :
+  const std::vector< std::size_t >& t_triinpoel ) :
```

Comment:
Yeah sure. Only reason I put "t_" in front of it is to clearly discriminate it from m_triinpoel, since the former is for the entire mesh, and the latter is for only that chunk. I've made the change though.